lua/astrocommunity/pack/cpp/README.md

```diff
@@ -8,7 +8,7 @@ This plugin pack does the following:
 - Adds [clangd_extensions.nvim](https://github.com/p00f/clangd_extensions.nvim) for language specific tooling
 - Adds [cmake-tools.nvim](https://github.com/Civitasv/cmake-tools.nvim) for building and debugging
 
-tips: the offset_encondings of clangd will confilicts whit null-ls, so add the following file in your user_config.
```

Comment:
confilicts is also wrong. Could you fix that typo while you're at it? :)

Reply:
Hi, I've fixed it. Thank you.

Reply:
No it's still wrong.

Reply:
Please check [b79b368e82dcadca69bed7d983478ddcbbc38bf5](https://github.com/AstroNvim/astrocommunity/pull/594/commits/b79b368e82dcadca69bed7d983478ddcbbc38bf5) and the latest commit

Reply:
I saw those. It's still wrong. It should be conflict. Next time please double check :)